draft-ietf-modpod-group-processes.md

```diff
@@ -303,13 +303,14 @@ prescriptive. The moderator team is free to decide on these actions.
 The expectation is that the minimal action necessary to maintain the
 comity of a forum will be attempted.
 
-The moderator team is responsible to the IESG.  The IESG
-MAY create or designate a forum to facilitate discussion about
-moderation, and refer interested parties to that forum.  All actions
-taken by the moderator team SHALL be reported to the IESG.  All
-bans longer than fourteen (14) days SHALL be reported to the forum in
-which the person was banned, as well as to that person, and in the
-case of a broad ban, to the community in a manner decided by the IESG.
+The moderator team is responsible to the IESG.  The IESG MAY create or
+designate a forum to facilitate discussion about moderation, and refer
+interested parties to that forum.  All actions taken by the moderator
+team SHALL be reported to the IESG, as well as to the those against whom
```

Comment:
typo: should be "as well as to those" (lose "the")

Reply:
fixed.